### PR TITLE
adding benchmark table data and ecc data from account table

### PR DESCRIPTION
**Pull request by anthonygwalters**
# Pull Request

## Merge Pre-Check

- [ ] Create a Pull Request task on our
      [Data Warehouse](https://app.asana.com/0/35737884613515/1204991342481434) Asana project
- [ ] <kbd>Format</kbd> has been run on all requested files
- [ ] `dbt build` successfully run for the entire project[^1]

[^1]:
    If only requesting 1 file, it's acceptable to run build only for that and its downstream
    dependencies.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205770546208249

### CODE REVIEW

**Review by CGibson17 — DISMISSED** [review]

File: src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__kfwd_dashboard.sql [hunk 1]
```diff
@@ -110,6 +110,48 @@ with
...
+        from
+            (
+                select
+                    {{
+                        teamster_utils.date_to_fiscal_year(
+                            date_field="benchmark_date",
+                            start_month=7,
+                            year_source="start",
+                        )
+                    }} as academic_year,
```
Comment (cbini) [inline src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__kfwd_dashboard.sql:141]: refactor to stg_kippadb__college_persistence

**Review by cbini — CHANGES_REQUESTED** [review]
@anthonygwalters I'll make these changes, just wanted to mark it up here

File: src/dbt/kipptaf/models/kippadb/base/base_kippadb__application.sql [hunk 2]
```diff
@@ -65,6 +65,7 @@ with
 
             acc.name as account_name,
             acc.type as account_type,
+            acc.adjusted_6_year_minority_graduation_rate,
```
Comment (cbini) [inline src/dbt/kipptaf/models/kippadb/base/base_kippadb__application.sql:68]: @anthonygwalters did you intend to use this in the dashboard view or just adding for other purposes?

Comment (anthonygwalters) [reply src/dbt/kipptaf/models/kippadb/base/base_kippadb__application.sql:68]: Needed for ad-hoc stuff and dashboard eventually

**Review by cbini — APPROVED** [review]

**Pull request merged**